packages/zoe/src/cleanProposal.js

```diff
@@ -47,7 +47,9 @@ export const cleanKeywords = uncleanKeywordRecord => {
 
   // Assert all names are ascii identifiers starting with
   // an upper case letter.
-  keywords.forEach(assertKeywordName);
```

Comment:
I wish the rule allowed this but I don't think it's worth the effort for us to fix

Reply:
I agree that it's not worth changing anything about the `github/array-foreach` rule, but it _would_ be nice to use [`eslint curly: ["error", "multi-line"]`](https://eslint.org/docs/latest/rules/curly#multi-line):
```js
  for (const keyword of keywords) assertKeywordName(keyword);
```